### ISSUE TITLE
Looping over bundles with one element

### ISSUE THREAD

**Issue by sperezconesa** [datreant/datreant#177]
Hello,

I have a small suggestion that I think could improve the usability of datreant in workflows that can vary the number of members of a bundle. I normally have this kind of structures in my notebooks:
```python
for treant in my_bundle:
      process the treant
```
The problem is that if I have done some filtering of the bundle and I end up with a single treant in the bundle. This gives an error. I normally patch this problem by doing:
```python
for treant in [my_bundle[0]]:
      process the treant
```
But of course I have to modify again if I change the bundle size. Am I missing a better way to do this? If not, maybe there is a simple solution that can be implemented in the code so that the top code works regardless of size?

Thanks a lot for your help and keep up the good work!
Best,
Sergio

**Comment by kain88-de** [comment]
Sorry for the late reply. Which version are you running? I cannot reproduce this from the current development branch.

To test I created a bundle with only two treants and selected a single one with boolean selection, everything boils down to boolean selection internally.

```python
import datreant as dtr
t1 = dtr.Treant('a')
t2 = dtr.Treant('aa')
b = dtr.Bundle(t1,t2)
for c in b[[True, False]]:
    print('xxx')
```

**Comment by sperezconesa** [comment]
Ahh. I think I missunderstood how bundles work because this doesn't work for me either:
```python
import datreant as dtr
t1 = dtr.Treant('a')
t2 = dtr.Treant('aa')
b = dtr.Bundle(t1,t2)
for c in b[0]:
    print('xxx')
```
But this does:
```
import datreant as dtr
t1 = dtr.Treant('a')
t2 = dtr.Treant('aa')
b = dtr.Bundle(t1,t2)
for c in b[[0]]:
    print('xxx')
```

**Comment by kain88-de** [comment]
Yeah with the first syntax you are requesting element 0 which is a treant.
The later selects a bundle containing a single element. As long as you
always give the getitem [] operator a list like object you will get a
bundle back.

Does this solve your issues?
